src/Unity.Mathematics/Noise/psrdnoise2D.cs

```diff
@@ -346,20 +352,24 @@ public static float3 srdnoise(float2 pos, float rot)
             return 11.0f * float3(n, dn0 + dn1 + dn2);
         }
 
-        //
-        // 2-D non-tiling simplex noise with fixed gradients and analytical derivative.
-        // This function is implemented as a wrapper to "srdnoise",
-        // at the math.minimal math.cost of three extra additions.
-        //
+        /// <summary>
+        /// 2-D non-tiling simplex noise with fixed gradients and analytical derivative.
+        /// This function is implemented as a wrapper to "srdnoise",
+        /// at the math.minimal math.cost of three extra additions.
+        /// </summary>
```

Comment:
Similar issue as `cellular2x2`, summary should be one liner and the rest in remarks. Recommend keeping the first sentence as summary and the rest as remarks.